samples/cellular/nrf_cloud_multi_service/sample.yaml

```
@@ -94,7 +94,7 @@ tests:
     integration_platforms:
       - nrf5340dk/nrf5340/cpuapp/ns
     platform_allow: nrf5340dk/nrf5340/cpuapp/ns
-    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
+    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek_nrf7000
```

```suggestion
    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
```
@krish2718 you may have not seen @glarsennordic 's comments about this. Please confirm if we are using Wi-Fi for the full network stack and not only scanning that this line should not have been changed to `nrf7002ek_nrf7000`.

---

I did see them, but haven't responded, yes, I will double check this and if its for full Wi-Fi then this change isn't needed, I will fix this thanks.